DEFINITIONS.md

```diff
@@ -0,0 +1,142 @@
+# QG Terms and Key Word Definitions
+Definitions of frequently-used terms within the QueerGlobal project. A developer may encounter some of these words or terms in various literature pertaining to the tools used by the application. Every term's definition comes from a summary pulled from Slack conversations, from ChatGPT, or from GitHub Copilot.
+
+
+## API
+Application Program Interface. Ex. REST API
+
+## Architecture
+The overall design of a software system, and the software components it is comprised of, the externally visible properties of those components, and the relationships among them. Ex. Object-oriented programming
+
+## Bug
+Any error or defect that cause the software/application or hardware to malfunction. Ex. QueerGlobal being spelled "QueerBlogal" in the app.
+
+## Chore (Pull Request Label)
+The presence of new/changed code that is: 
+1) not included in the feature requirements 
+2) does not meet the required workflow, process or function point
+Ex. Changing the Typescript type of a "resource" variable value from object to string
+
+## Cloud
+A "cloud" is just a network of servers, each with a unique function. Some servers use computing power to run applications or "deliver a service." Others are responsible for storing data or files. Ex. Google Drive
+
+## Cloud Native
+Building and running applications in a way that fully utilizes the benefits provided by cloud environments. It involves developing applications as a collection of loosely coupled, independently deployable services, often referred to as microservices, which can be scaled independently based on demand. Ex. iCloud storage
+
+## Container
+A code sandbox; A standard unit of software that packages up code and all its dependencies so the application runs quickly and reliably from one computing environment to another. It is a unit of a runnable application inside of an image. Please see Image (Container) for more information. A holder of program images (please see Images for more information.), used for transport. Ex. A Docker container of the nginX image
```

Comment:
Is it worth having related terms at the end of each term, like "see Docker", with a link to Docker definition?

Reply:
Also, nginx has that capital X at the end.